audio_extraction/audio_extraction.ipynb

```diff
@@ -0,0 +1,245 @@
+    "                        ## ffmpeg -i input.mp4 -vn -q:a 0 -map a audio.mp3\n",
+    "                        ## '-i' is the input, '-vn' excludes video,\n",
+    "                        ## '-q:a' is the mp3 encoding,\n",
+    "                        ## '0' is the best quality mp3 encoding,\n",
+    "                        ## '-map a' only grabs audio\n",
+    "                        \n",
+    "                        # Using ffmpeg to transform the video file into a .mp3 file\n",
+    "                        try:\n",
+    "                            out, error_message = ffmpeg.input(str(child)) \\\n",
+    "                            .output(filename = str(audio_file_path),\n",
+    "                                   **{'qscale:a': 0,\n",
+    "                                      'map': 'a'}) \\\n",
+    "                            .overwrite_output() \\\n",
+    "                            .run()\n",
+    "                        # Print the error message if the try doesn't work\n",
+    "                        except ffmpeg.Error as error_message:\n",
+    "                            print('ffmpeg stderr:', error_message.stderr())\n",
+    "                            raise error_message\n",
+    "                        \n",
+    "                        # Incremenet the counter for the number of video files processed\n",
+    "                        audio_file_counter += 1\n",
+    "    # Return a print message that states how many audio files were extracted\n",
+    "    if (audio_file_counter == 1):\n",
+    "        file_word = 'file'\n",
+    "    else:\n",
+    "        file_word = 'files'\n",
+    "    # Print how many audio files were extracted\n",
+    "    return (str(audio_file_counter) + ' audio ' + file_word + ' extracted!')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Video path:  /Users/Jackie/Documents/Coding/jupyter nbs (python)/audio_extraction/data\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "'2 audio files extracted!'"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Testing function\n",
+    "# Make the path to the data folder that contains the video files\n",
+    "v_path_1 = Path.cwd().joinpath('data')\n",
+    "# Make a directory that you want the extracted audio to go into\n",
+    "a_path_1 = Path.cwd().joinpath('extracted_audio')\n",
+    "# Show where the video files are coming from\n",
+    "print(\"Video path: \", v_path_1)\n",
+    "# Test the function\n",
+    "extract_audio_from_video(video_path = v_path_1, to_folder = a_path_1, log_flag = True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Note about logging:\n",
```

Comment:
put this in the README when it is done, but I suggested another way to do the logging

Reply:
I've removed these comments as I've changed the logging system to the system I've mentioned above regarding unique file names and minutes.